src/app/member/components/MemberActionRender.tsx

```diff
@@ -1,14 +1,22 @@
-import ActionsMenu from '@/components/common/ActionsMenu';
+import ActionsMenu from '@/common/ActionMenu/ActionsMenu';
+import { MemberRole } from '@/lib/routes';
 import { Info, Trash2, Edit, Send } from "lucide-react";
 import React from 'react';
 
-// When we will implement in table we will update this type
-type Props ={
+interface MemberInfo {
+  id: number;
+  name: string;
+  phone: string;
+  shift: string;
```

Comment:
can we use shift_enums and role_enums instead of string  in role and shift property

Reply:
We don't actually shift here actually. Maybe remove that?